src/manifest/owned/pfm.rs

```diff
@@ -259,14 +259,19 @@ impl owned::Element for Element {
 }
 
 impl<'f, F: 'f + Flash> owned::FromUnowned<'f, F> for Element {
-    type Unowned = manifest::pfm::Pfm<'f, F, provenance::Adhoc>;
+    type Manifest = manifest::pfm::Pfm;
 
     fn from_container(
-        container: manifest::Container<'f, Self::Unowned, F, provenance::Adhoc>,
+        container: manifest::Container<
+            'f,
+            Self::Manifest,
+            F,
+            provenance::Adhoc,
+        >,
```

Comment:
This isn't an actionable comment, but I don't like what `rustfmt` did here.

Reply:
Same. I actually tried to throw a #[rustfmt::skip]` on the `foo: Stuff<...>` binding and it was ignored. I can clean this up and rustfmt::skip the whole function decl if you'd prefer.

Reply:
No, let's leave it as is rather than `rustfmt::skip`-ing a larger chunk of code.